src/plugins/mod_chat_history_sqlite.c

```diff
@@ -142,7 +142,7 @@ void user_login(struct plugin_handle* plugin, struct plugin_user* user)
 	struct cbuffer* buf = NULL;
 	struct linked_list* found = (struct linked_list*) list_create();
 	
-	sql_execute(data, get_messages_callback, found, "SELECT * FROM chat_history ORDER BY time DESC LIMIT 0,%d;", (int) data->history_connect);
+	sql_execute(data, get_messages_callback, found, "SELECT from_nick,message, datetime(time, 'localtime') as time FROM chat_history ORDER BY time DESC LIMIT 0,%d;", (int) data->history_connect);
```

Comment:
Hard to argue what the "correct behavior" is, given that the underlying protocol has no concept of time zones.
I think it is better to use relative time ultimately for chat history - e.g. message happened "2 days ago" or "1 minute ago".